### DISCUSSION TITLE
Error in timeline photo classification after upload

### DISCUSSION THREAD

**Discussion by olivier-da** [immich-app/immich#5088] [Q&A]
Hello,

I get bad classification in the timeline for photos uploaded from my mobile. 
It's strange because for photos taken by the same mobile and the same day they are not classified the same day. The common thing is for all photos it's not the right date.

I don't see any issues in EXIF metada and the docker logs don't show any errors.
I miss some idéas to analyze.

Examples:

This photo IMG_20231031_183744.jpg taken 31st october 2023, it's classify at 14th November 2023 (2 days ago)
exif extract:
![image](https://github.com/immich-app/immich/assets/151080159/e856f0ea-68e3-4013-8e42-6290f5e0e2c7)

This 2nd photo IMG_20231031_185548.jpg taken the same date by the same device  is classify 12nd November 2023 (4 days ago)
EXIF extract
![image](https://github.com/immich-app/immich/assets/151080159/dea3f9cc-afdc-4b92-b792-4b5f3e1f5f61)

Someone could help me to understand the issue?

Thank you

**Comment by alextran1502** [comment]
Would you be able to share the photo for troubleshooting?

**Comment by olivier-da** [comment]
I close the discussion since I have other issues during uploading where some photos are not uploaded with the error
 _Unable to run job handler (clipEncoding/clip-encode): TypeError: fetch failed_

I think I'll come back to Immich when it would be more stable.
BTW thx a lot to have this initiative which clearly answers a need
